transport-native-unix-common/src/main/java/io/netty/channel/unix/DomainSocketChannelConfig.java

```diff
@@ -77,4 +79,24 @@
      * Return the {@link DomainSocketReadMode} for the channel.
      */
     DomainSocketReadMode getReadMode();
```

Comment:
We can’t make these changes in a point release :( That said we can keep the ones on the actual implementation. Please revert this part

Reply:
removed. is there somewhere you'd like to track adding this in a future release?